src/main/java/com/sumi/smartemployeeservice/services/EmployeeService.java

```diff
@@ -0,0 +1,60 @@
+package com.sumi.smartemployeeservice.services;
+
+import com.sumi.smartemployeeservice.modal.Employee;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+public class EmployeeService {
+
+    private List<Employee> employees = new ArrayList<>(Arrays.asList(
+            new Employee(1, "sumi", "katu"),
+
+            new Employee(2, "sumi", "katu")
+
+    ));
+
+
+    public List<Employee> getEmployees() {
+
+        return employees;
+    }
+
+    public Employee getEmployee(int id) {
+        Employee emp = new Employee();
+        for (Employee employee : employees
+        ) {
+            if (employee.getId() == id) {
+                emp = employee;
+            }
+        }
+        return emp;
+    }
```

Comment:
Now you should get and save employess to database.

Reply:
will do soon